tests/SIL.LCModel.Tests/DomainServices/WritingSystemServicesTests.cs

```diff
@@ -255,6 +255,34 @@ public void UpdateWritingSystemListField_RemovesMergedCodeAfterMergeWith()
 
 		/// <summary/>
 		[Test]
+		public void DeleteWritingSystem()
+		{
+			int m_wsEn = Cache.WritingSystemFactory.GetWsFromStr("en");
+			int m_wsFr = Cache.WritingSystemFactory.GetWsFromStr("fr");
+
+			CoreWritingSystemDefinition enBlz;
+			WritingSystemServices.FindOrCreateWritingSystem(Cache, null, "blz", false, false, out enBlz);
+
+			var revIndex = Cache.ServiceLocator.GetInstance<IReversalIndexRepository>().FindOrCreateIndexForWs(m_wsEn);
```

Comment:
To make this test simplere let us create the reversal index for wsBlz instead of m_wsEn

Reply:
Yes, I created reversal index for wsBlz.